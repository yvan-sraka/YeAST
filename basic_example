#! /usr/bin/env yeast
#! python

print("Hello, World!")

!##! ruby

puts "Hello,#! node
console.log(" World!\"");!#
