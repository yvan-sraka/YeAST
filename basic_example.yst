#! /usr/bin/env yeast
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/!\ DON'T DO THIS AT HOME OR WORK
THIS IS FOR PEDAGOGICAL PURPOSE ONLY
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#! python
print("Hi,")

!##! ruby
puts "Do you want some #! node
process.stdout.write("yeast");!# in your pasta?"
