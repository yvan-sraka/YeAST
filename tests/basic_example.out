Hello, World!
What's, Up!
